src/renderer.rs

```diff
@@ -267,26 +268,37 @@ fn version_footer() -> Markup {
     }
 }
 
-fn wget_footer(title_path: &str, current_user: Option<&CurrentUser>) -> Markup {
-    let count = {
-        let count_slashes = title_path.matches('/').count();
-        if count_slashes > 0 {
-            count_slashes - 1
-        } else {
-            0
-        }
+fn wget_footer(abs_path: &Uri, root_dir_name: Option<&str>, current_user: Option<&str>) -> Markup {
+    let escape_apostrophes = |x: &str| x.replace('\'', "'\"'\"'");
```

Comment:
Let's not do the JS fat arrow function thing here. Why not just make this a proper little inner function instead?